lnwallet/channel.go

```diff
@@ -101,6 +91,24 @@ var (
 		"state data loss")
 )
 
+// ErrCommitSyncLocalDataLoss is returned in the case that we receive a valid
+// commit secret within the ChannelReestablish message from the remote node AND
+// they advertise a RemoteCommitTailHeight higher than our current known
+// height. This means we have lost some critical data, and must fail the
+// channel and MUST NOT force close it. Instead we should wait for the remote
+// to force close it, such that we can attempt to sweep our funds. The
+// commitment point needed to sweep the remote's force close is encapsuled.
+type ErrCommitSyncLocalDataLoss struct {
+	// CommitPoint is the last unrevoked commit point, sent to us by the
+	// remote when we determined we had lost state.
+	CommitPoint *btcec.PublicKey
+}
+
+func (e *ErrCommitSyncLocalDataLoss) Error() string {
```

Comment:
Missing `godoc` comment. Can also log the serialized commit point in this error as well, and maybe also the channel point while we're at it?

Reply:
Done